### DISCUSSION TITLE
New hardware: netbook Acer Aspire One 522

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#31] [Show and tell]
Wav: ko
Touchpad: ko
keyboard: ok
usb mouse: ok
ethernet: ko
VBE: ok but not max resolution 
[aceraspireone522_u13.txt](https://github.com/VendelinSlezak/BleskOS/files/13746577/aceraspireone522_u13.txt)

**Comment by VendelinSlezak** [comment]
> Wav: ko

Update 14 should be able to play your WAV.

> Touchpad: ko

Do you have touchpad allowed in BIOS? Because according to log, BleskOS recognized and successfully initalized it:
`second channel test successfull`
`second channel device ID: 0x00`

> ethernet: ko

I see you have Qualcomm Atheros card. I also have one computer with Qualcomm Atheros ethernet card, so it is definitely on my list of wanted drivers. I hope one day I will write it.

**Comment by defdefred** [comment]
Wav: ok but no sound (I should try an headset)
Touchpad: ok (it was loked by an fn key)
[aceraspireone522u14.txt](https://github.com/VendelinSlezak/BleskOS/files/13760446/aceraspireone522u14.txt)

**Comment by VendelinSlezak** [comment]
> Wav: ok but no sound (I should try an headset)

I have one computer with audio codec 0x14F15066 and in Update 15 I finally got sound, your computer have audio codec 0x14F15068 so we will see if it will work.

**Comment by defdefred** [comment]
Tried with u15 and wav is working!

Also tried more than 800x600 graphics to match the nice 1280x720 screen but I got error and red screen.
VBE don't have proper 16/9 resolution.

**Comment by VendelinSlezak** [comment]
> Tried with u15 and wav is working!

Great :)

> Also tried more than 800x600 graphics to match the nice 1280x720 screen but I got error and red screen.

Unfortunately VBE often do not have best resolution. However if there is option to select some 1280x720 mode, I am curious what went wrong. Did you select mode with 24/32 bits per color? Can you please post photo of screen when you manually selected 1280x720 mode in bootloader?
